tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -98,7 +92,8 @@ class MklFusedBatchNormFwdPrimitive : public MklPrimitive {
   //   mean_data:     output data buffer of means
   //   variance_data: output data buffer of variances
   void Execute(const T* src_data, const U* weights_data, T* dst_data,
-               U* mean_data, U* variance_data, U* workspace_data) {
+               U* mean_data, U* variance_data,
+               std::shared_ptr<stream> fwd_stream, U* workspace_data) {
```

Comment:
Just curious, why do we put `fwd_stream` in front of `workspace_data`? (`BwdPrimitive` and other PRs have `stream` last.)

Reply:
Right. This conflicted with another PR that was merged recently. So I decided to put it this way. Originally fwd_stream was the last.